06_modules_lab/03.py

```diff
@@ -8,4 +8,35 @@
 
 - Take threshold and path as command line arguments
 """
+import os
+import readchar
+import sys
 
+correct_input = False
+if len(sys.argv)==3:
+    current_working_directory = sys.argv[1]
+    MinimumSizeInMB = float(sys.argv[2])
```

Comment:
Use lowercase only variable names (`maximum_size_in_mb` is good, `threshold` is even better)